code/__DEFINES/misc.dm

```diff
@@ -458,3 +457,7 @@ var/global/list/ghost_others_options = list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 #define SHELTER_DEPLOY_BAD_AREA "bad area"
 #define SHELTER_DEPLOY_ANCHORED_OBJECTS "anchored objects"
 
+//debug printing macros
+#define debug_world(msg) if (Debug2) world << "DEBUG: [msg]"
+#define debug_admins(msg) if (Debug2) admins << "DEBUG: [msg]"
```

Comment:
yes, you can use a list in <<.

in fact, i think you can use this for sending client images to avoid the user code level loop, just do list_of_peeps_to_get_client_image << image

you might even be able to do it with a list of images.